handler/pullRequestHandler_test.go

```diff
@@ -124,3 +124,81 @@ func Test_hasDescription(t *testing.T) {
 		}
 	}
 }
+
+func Test_isAnonymous(t *testing.T) {
+	var anonymousSignOpts = []struct {
+		title        string
+		message      string
+		expectedBool bool
+	}{
+		{
+			title:        "Correctly signed off full string",
+			message:      "Signed-off-by: openfaas",
+			expectedBool: false,
+		},
+		{
+			title:        "Correctly signed off within string",
+			message:      "This PR was Signed-off-by: rgee0",
+			expectedBool: false,
+		},
+		{
+			title:        "Signed off with an anonymous email",
+			message:      "Signed-off-by: users@noreply.github.com",
+			expectedBool: true,
+		},
+	}
+	for _, test := range anonymousSignOpts {
+		t.Run(test.title, func(t *testing.T) {
+
+			anonymousSign := isAnonymousSign(test.message)
+
+			if anonymousSign != test.expectedBool {
+				t.Errorf("Is anonymous sign - Testing '%s'  - wanted: %t, found %t", test.message, test.expectedBool, anonymousSign)
+			}
+		})
+	}
+}
+
+func Test_hasAnonymousSign(t *testing.T) {
+	var anonymousSignOpts = []struct {
+		title    string
+		commits  []*github.RepositoryCommit
+		expected bool
+	}{
+		{
```

Comment:
There is a third use-case here, where there isn't any sign-off message. Let's add it?

Reply:
You are right, I'll add it